master/internal/resourcemanagers/scheduler_config.go

```diff
@@ -91,6 +91,20 @@ func (s *SchedulerConfig) GetType() string {
 	}
 }
 
+// GetPreemption returns whether the scheduler is set to preempt.
+func (s *SchedulerConfig) GetPreemption() bool {
```

Comment:
`GetPreemption` -> `PreemptionEnabled` i think sounds better

Reply:
that is better I can update it at once with the other comments